### DISCUSSION TITLE
Laravel backpack framework admin and user login

### DISCUSSION THREAD

**Discussion by ondrejmlcak** [Laravel-Backpack/community-forum#1250] [Q&A (Help)] [answered]
Hi, I am using the Backpack framework in my Laravel project. And I want to ask how to make login and registration for both admins and users. To keep it separate.

[a comment by an automated account is not shown]

**Comment by karandatwani92** [comment]
Hey @ondrejmlcak

I hope the bot's answer helped you figure out the solution.

Closing it for now - Feel free to reopen if you don't find the answer satisfactory.
